README.md

```diff
@@ -14,6 +17,24 @@ This repo contains the code for the official website of Zolplay.
 - [pnpm](https://pnpm.io/)
 
 ## Set up local environment
+
+copy `.env.example` -> `.env`
+
+```env
+NEXT_PUBLIC_OG_URL=https://YOUR-DOMAIN.com/api/og
```

Comment:
On second thought, this should be 
```suggestion
NEXT_PUBLIC_OG_URL=http://localhost:3000/api/og
```
Since we're writing in the local setup section here.